website/events/helpers/models.py

```diff
@@ -1,9 +1,14 @@
+# for local setup 
```

Comment:
All dependencies are declared in `config/requirements.txt`, used by the Dockerfile that prepares the environment, they should not be duplicated on each `.py`...